### ISSUE TITLE
Set App Volume StreamDeck+ Dial support

### ISSUE THREAD

**Issue by sejbr** [GeekyEggo/SoundDeck#108]
Hi, love the plugin.
Any chance of implementing app volume controls for the new StreamDeck+ dials?
Currently there are native volume controls for the SD+ but it can only be set to either all media, or specific processes.
No way to set it to active foreground application. The dials also seem to need specific implementation. I don't see a way to bind existing key action to dials. Only keyboard macros.

**Comment by GeekyEggo** [comment]
Hey @sejbr, so this _is_ on the roadmap for Sound Deck, but at the moment it’s priority is lower as Volume Controller offers very similar functionality, albeit with some missing features, e.g. foreground app (as you mentioned).

I’m currently hoping to release Stream Deck+ support for some key features first, and then add this shortly afterwards.

**Comment by NinjaNSR** [comment]
I agree with this but I would LOVE if you guys allowed me to bind specific devices to my dials. That would allow me to have a chat/game mix without having to bind it to the actual buttons. I currently can't find any plugin that has this other than you guys.
